sdk/core/abort-controller/README.md

```diff
@@ -41,7 +41,7 @@ doAsyncWork({ abortSignal: signal });
 ### Example 3 - Aborting sub-tasks
 
 ```js
-import { Aborter } from "@azure/core-aborter";
+import { Aborter } from "@azure/abort-controller";
 
 const allTasksSignal = new Aborter();
```

Comment:
I know this isn't part of your change, but I think this example was missed when changing to the new API.

Reply:
All 3 examples are stale right?

Reply:
Yes, the `abortSignal` part is right, but not the method to get the `AbortSignal`.